src/views/village/content/MaterialsCard.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <el-container>
+        <el-row>
+
+            <el-col :span="6">
+                <el-image :src="require('@/assets/image/materials/wood.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="material-image"
+                ></el-image>
+                {{materials.wood}}
+            </el-col>
+
+            <el-col :span="6">
+                <el-image :src="require('@/assets/image/materials/clay.png')"
+                          :fit="'scale-down'"
+                          alt="Clay"
+                          class="material-image"
+                ></el-image>
+                {{materials.clay}}
+            </el-col>
+
+            <el-col :span="6">
+                <el-image :src="require('@/assets/image/materials/iron.png')"
+                          :fit="'scale-down'"
+                          alt="Iron"
+                          class="material-image"
+                ></el-image>
+                {{materials.iron}}
+            </el-col>
+
+            <el-col :span="6">
+                <el-image :src="require('@/assets/image/materials/villagers.jpg')"
+                          :fit="'scale-down'"
+                          alt="Iron"
+                          class="material-image"
+                ></el-image>
+                {{materials.villagers}}
+            </el-col>
```

Comment:
You can make an array of MaterialType and ImagePath objects, and then use v-for to loop over them.